[2022-05-13 21:47:32.251] [remoteagent] [info] Extension host agent started.
[2022-05-13 21:47:33.543] [remoteagent] [info] [127.0.0.1][70ced270][ManagementConnection] New connection established.
[2022-05-13 21:47:33.779] [remoteagent] [info] [127.0.0.1][27d4d5d5][ExtensionHostConnection] New connection established.
[2022-05-13 21:47:33.784] [remoteagent] [info] [127.0.0.1][27d4d5d5][ExtensionHostConnection] <85878> Launched Extension Host Process.
[2022-05-13 21:47:41.682] [remoteagent] [info] Getting Manifest... ms-python.python
[2022-05-13 21:47:41.682] [remoteagent] [info] Getting Manifest... ms-python.vscode-pylance
[2022-05-13 21:47:41.683] [remoteagent] [info] Getting Manifest... ms-toolsai.jupyter
[2022-05-13 21:47:41.852] [remoteagent] [info] Installing extension: ms-python.vscode-pylance
[2022-05-13 21:47:41.856] [remoteagent] [info] Installing extension: ms-python.python
[2022-05-13 21:47:41.935] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateIcon (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-05-13 21:47:41.936] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateTitle (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-05-13 21:47:42.558] [remoteagent] [info] Installing extension: ms-toolsai.jupyter
[2022-05-13 21:47:46.562] [remoteagent] [info] Downloaded extension: ms-python.python /home/ubuntu/.vscode-server/data/CachedExtensionVSIXs/ms-python.python-2022.6.2
[2022-05-13 21:47:46.744] [remoteagent] [info] Downloaded extension: ms-toolsai.jupyter /home/ubuntu/.vscode-server/data/CachedExtensionVSIXs/ms-toolsai.jupyter-2022.4.1021342353
[2022-05-13 21:47:46.749] [remoteagent] [info] Downloaded extension: ms-python.vscode-pylance /home/ubuntu/.vscode-server/data/CachedExtensionVSIXs/ms-python.vscode-pylance-2022.5.1
[2022-05-13 21:47:52.629] [remoteagent] [info] Extracted extension to /home/ubuntu/.vscode-server/extensions/.4bb475a2-2c2a-466b-bb29-3f6a0eaf4015: ms-toolsai.jupyter
[2022-05-13 21:47:52.646] [remoteagent] [info] Renamed to /home/ubuntu/.vscode-server/extensions/ms-toolsai.jupyter-2022.4.1021342353
[2022-05-13 21:47:52.734] [remoteagent] [info] Extracting completed. ms-toolsai.jupyter
[2022-05-13 21:47:52.743] [remoteagent] [info] Extension installed successfully: ms-toolsai.jupyter
[2022-05-13 21:48:04.657] [remoteagent] [info] Extracted extension to /home/ubuntu/.vscode-server/extensions/.0703a301-08de-4c20-93ec-f3f4145e1a05: ms-python.python
[2022-05-13 21:48:04.740] [remoteagent] [info] Renamed to /home/ubuntu/.vscode-server/extensions/ms-python.python-2022.6.2
[2022-05-13 21:48:04.758] [remoteagent] [info] Extracting completed. ms-python.python
[2022-05-13 21:48:04.835] [remoteagent] [info] Extension installed successfully: ms-python.python
[2022-05-13 21:48:13.442] [remoteagent] [info] Extracted extension to /home/ubuntu/.vscode-server/extensions/.a90ec90d-c148-4f0f-a2ad-9fe6fa0deb70: ms-python.vscode-pylance
[2022-05-13 21:48:13.637] [remoteagent] [info] Renamed to /home/ubuntu/.vscode-server/extensions/ms-python.vscode-pylance-2022.5.1
[2022-05-13 21:48:13.639] [remoteagent] [info] Extracting completed. ms-python.vscode-pylance
[2022-05-13 21:48:13.741] [remoteagent] [info] Extension installed successfully: ms-python.vscode-pylance
[2022-05-13 21:48:53.236] [remoteagent] [info] [127.0.0.1][70ced270][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-05-13 21:48:53.254] [remoteagent] [info] [127.0.0.1][27d4d5d5][ExtensionHostConnection] <85878> Extension Host Process exited with code: 0, signal: null.
[2022-05-13 21:48:53.255] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-05-13 21:49:18.336] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-05-13 21:49:18.336] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-05-13 21:49:20.381] [remoteagent] [info] [127.0.0.1][c3df794c][ManagementConnection] New connection established.
[2022-05-13 21:49:20.832] [remoteagent] [info] [127.0.0.1][97982ac3][ExtensionHostConnection] New connection established.
[2022-05-13 21:49:20.849] [remoteagent] [info] [127.0.0.1][97982ac3][ExtensionHostConnection] <86036> Launched Extension Host Process.
[2022-05-13 21:49:42.930] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateTitle (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-05-13 21:49:42.931] [remoteagent] [error] ["Error: Could not find pty on pty host","    at m._throwIfNoPty (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at m.updateIcon (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:13:8361)","    at g.onPromise (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4879)","    at g.onRawMessage (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:4267)","    at /home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:12:3555","    at D.invoke (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at N.deliver (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at h.fire (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.p (/home/ubuntu/.vscode-server/bin/da15b6fd3ef856477bf6f4fb29ba1b7af717770d/out/vs/platform/terminal/node/ptyHostMain.js:6:34973)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-05-13 21:54:18.337] [remoteagent] [info] New EH opened, aborting shutdown
[2022-05-13 23:09:32.304] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/ubuntu/buenComensalProject)
[2022-05-13 23:09:32.304] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[2022-05-13 23:12:13.318] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/ubuntu/buenComensalProject)
[2022-05-13 23:12:13.318] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[2022-05-13 23:12:13.657] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/ubuntu/buenComensalProject)
[2022-05-13 23:12:13.657] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[2022-05-14 02:45:10.902] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/ubuntu/buenComensalProject)
[2022-05-14 02:45:10.905] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[2022-05-14 04:57:19.385] [remoteagent] [info] [127.0.0.1][c3df794c][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-05-14 04:57:19.444] [remoteagent] [info] [127.0.0.1][97982ac3][ExtensionHostConnection] <86036> Extension Host Process exited with code: 0, signal: null.
[2022-05-14 04:57:19.444] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-05-14 05:02:19.445] [remoteagent] [info] Last EH closed, shutting down
